ReactNativeClient/lib/models/Note.js

```diff
@@ -342,8 +361,13 @@ class Note extends BaseItem {
 			tempOptions.conditions = cond;
 			if ('limit' in tempOptions) tempOptions.limit -= uncompletedTodos.length;
 			const theRest = await this.search(tempOptions);
+			const items = uncompletedTodos.concat(theRest);
+
+			for (const item of items) {
```

Comment:
We can't load the preview here. As discussed in the forum, the preview should be loaded dynamically, and only for the currently visible notes, for performance reasons.

Reply:
Is there currently a way to identify the notes which are visible in the notelist, or will that have to be implemented?